src/css/projects.css

```diff
@@ -0,0 +1,45 @@
+.projects-page {
+  margin-top: 75px !important;
```

Comment:
Do you need this !important?

```suggestion
  margin-top: 12px;
```

Reply:
Also set to 12px to match the MemberInfoContainer margin.